### PR TITLE
Fix: 'python' highlighting when not a keyword

### PR DESCRIPTION
**Pull request by idillon-sfl**
Before:
Screenshot 2024-11-15 at 10 19 12 AM

After:
Screenshot 2024-11-15 at 10 19 44 AM

### CODE REVIEW

File: client/test/grammars/test-cases/functions.bb [hunk 1]
```diff
@@ -25,8 +25,8 @@
 >    else:
 >        return "dependency"
  
->python () {
-#^^^^^^ source.bb storage.type.function.python.bb - entity.name.function.python.bb
+python () {
+#<------ source.bb storage.type.function.python.bb - entity.name.function.python.bb
```
Comment (idillon-sfl) [inline client/test/grammars/test-cases/functions.bb:29]: vscode-tmgrammar-test was including `>` into the match, which caused the test to fail

**Comment by idillon-sfl** [comment]
Great, we had tests! Good job @WilsonZiweiWang 

Sorry for prematurely calling everyone for review (again). It should work now.

File: client/test/grammars/test-cases/functions.bb (hunk 1 above)
Comment (WilsonZiweiWang) [reply client/test/grammars/test-cases/functions.bb:29]: Adding a `>` at the beginning makes it easier to align with the expected scopes below, but it did pose a risk of causing the test to fail in a case like this.

**Review by WilsonZiweiWang — APPROVED** [review]
LGTM

**Pull request merged**